template/web/app/paymaster-bundler/_utils/fetchNFTs.ts

```diff
@@ -0,0 +1,60 @@
+import { SmartAccountClient } from 'permissionless';
+import { PublicClient } from 'viem';
+
+import { nftAbi } from '../_components/abi';
+import { NFTType } from '../types';
+
+export default async function fetchNFTs(smartAccount: SmartAccountClient, client: PublicClient) {
+  // Get # of NFTs owned by address
+  const address = smartAccount.account?.address;
+
+  const numTokens = await client.readContract({
+    address: '0x66519FCAee1Ed65bc9e0aCc25cCD900668D3eD49',
```

Comment:
In the follow up PR let's rewrite this, and use `generateContractHook`. cc @arsood